lib/tests/streamlit/streamlit_test.py

```diff
@@ -425,17 +433,6 @@ def test_st_help(self):
             el.doc_string.doc_string.startswith("Display text in header formatting.")
         )
         self.assertEqual(el.doc_string.type, "<class 'method'>")
-        if sys.version_info < (3, 9):
-            # Python < 3.9 represents the signature slightly differently
-            self.assertEqual(
-                el.doc_string.signature,
-                "(body: object, anchor: Union[str, NoneType] = None) -> 'DeltaGenerator'",
-            )
-        else:
-            self.assertEqual(
-                el.doc_string.signature,
-                "(body: object, anchor: Optional[str] = None) -> 'DeltaGenerator'",
-            )
```

Comment:
For my understanding & also documentation, could you elaborate on the removal of this test logic?

Reply:
It was odd for me to test the signature, but now I've updated the test instead of removing it 👍